src/components/previewCard/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+import UploadImagesIcon from "../icons/uploadImage"
+import ArrowRightTeal from "../icons/arrowRight"
+import { FixedCard } from "../card/fixed"
+
+interface Props {
+  name: string
+  location: {
+    address?: string
+    zipCode: string
+    city?: string
+  }
+  title?: string
+  previewLabel: string
+  image?: string
+  logo?: string
+  link?: string
+  editMode?: boolean
+  handleOnClick?: () => void
+  handleOnError?: (e) => void
+  handleOnClickText?: () => void
+}
+
+const PreviewCard: FC<Props> = ({
+  name,
+  location,
+  title,
+  previewLabel,
+  image,
+  logo,
+  link,
+  editMode,
+  handleOnClick,
+  handleOnError,
+  handleOnClickText,
+}) => {
+  return (
+    <FixedCard
+      renderImage={() => (
+        <div className="bg-grey-1 flex justify-center items-center text-grey-2 md:h-cardImage w-12/12 md:w-card">
+          {!image && editMode ? (
+            <UploadImagesIcon width="72px" height="72px" />
```

Comment:
Instead of relying on two boolean props, I would prefer to pass the image with `image` explicitly

Reply:
we discussed to drop the `editMode` prop and just rely on either passing an image or not. as listings will always have an image, it should never show this upload icon

we didn't really find a viable solution to take everything apart nicely